hermes-console-vue/src/views/admin/consistency/inconsistent-topics-listing/InconsistentTopicsListing.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+  import { computed } from 'vue';
+  import { useI18n } from 'vue-i18n';
+  const { t } = useI18n();
+  const props = defineProps<{
+    inconsistentTopics: string[];
+    filter?: string;
+  }>();
+  const filteredTopics = computed(() => {
+    return props.inconsistentTopics.filter(
+      (topic) => !props.filter || topic.indexOf(props.filter) !== -1,
+    );
+  });
```

Comment:
I would suggest adding some empty lines to improve clarity.

We can also use `topic.includes(props.filter)` instead of `topic.indexOf(props.filter) !== -1`. I know `indexOf` was used in other component created by myself, but only recently I found out this could be simplified – for some reason I didn't know about `String.includes` in JS before🤦🏼‍♂️. (We can simplify it there as well!😄)

```suggestion
  const { t } = useI18n();
  
  const props = defineProps<{
    inconsistentTopics: string[];
    filter?: string;
  }>();
  
  const filteredTopics = computed(() => {
    return props.inconsistentTopics.filter(
      (topic) => !props.filter || topic.includes(props.filter),
    );
  });
```